app/presenters/registration_presenter.rb

```diff
@@ -1,12 +1,6 @@
 # frozen_string_literal: true
 
 class RegistrationPresenter < BaseRegistrationPresenter
-  def displayable_location
-    location = show_translation_or_filler(:location)
-
-    I18n.t(".registrations.show.business_information.labels.location", location: location)
-  end
```

Comment:
This is a left-over of previous development, I have implemented this method in the base presenter already, so we don't need this version any more, and with the shared partial it was causing errors for missing translations hence I had to delete it here.